ts/parts/OrdinalAxis.ts

```diff
@@ -918,9 +918,15 @@ addEvent(Chart, 'pan', function (this: Highcharts.Chart, e: any): void {
         xAxis = chart.xAxis[0],
         overscroll = xAxis.options.overscroll,
         chartX = e.originalEvent.chartX,
+        panning = (chart.options.chart as any).panning,
```

Comment:
Same as before. It could be replaced by `chart.options.chart && chart.options.panning`.